deploy/addons/heapster/heapster-rc.yaml

```diff
@@ -16,7 +15,7 @@ spec:
   template:
     metadata:
       labels:
-        kubernetes.io/cluster-service: 'true'
+        addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
I think this actually needs to go up on the RC labels, from looking here:
https://github.com/kubernetes/kubernetes/blob/b6b3ff59be40be0f907e75d4eebbae4246dbe81f/cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml